### PR TITLE
Add compatibility with Smush 3.0

### PR DESCRIPTION
**Pull request by av3nger**
Smush 3.0 removes globals. Please review this PR with proposed changes to support Smush 3.0

### CODE REVIEW

File: lib/classes/compatibility/wp-smush.php [hunk 1]
```diff
@@ -17,7 +17,7 @@ class WPSmush extends ICompatibility {
             protected $title = 'WP Smush';
             protected $constant = 'WP_STATELESS_COMPATIBILITY_WPSMUSH';
             protected $description = 'Ensures compatibility with WP Smush.';
-            protected $plugin_file = ['wp-smushit/wp-smush.php', 'wp-smush-pro/wp-smush.php', 'wp-smushit-pro/wp-smush-pro.php'];
```
Comment (alimuzzaman) [inline lib/classes/compatibility/wp-smush.php:20]: You shouldn't remove this.
> , 'wp-smushit-pro/wp-smush-pro.php'

**Comment by alimuzzaman** [comment]
Hi@av3nger,
Can you please make the pull request to the `latest` branch?
Thanks for the contribution.

File: lib/classes/compatibility/wp-smush.php (hunk 1 above)
Comment (av3nger) [reply lib/classes/compatibility/wp-smush.php:20]: Smush does not use such folder/file combination. Only:
wp-smushit/wp-smush.php - for wp.org version
wp-smush-pro/wp-smush.php - for Pro version
Anything else is not the original Smush plugin.

**Review by alimuzzaman — APPROVED** [review]
Looks good

**Pull request merged**